### ISSUE TITLE
Command 'ESLint: Restart ESLint Server' resulted in an error (command 'eslint.restart' not found)

### ISSUE THREAD

**Issue by chinese-words-separator** [microsoft/vscode-eslint#1473]
I tried to restart ESLint, and got that error

ESLint was running in my VSCode before, even without installing package.json in my project folder (a chrome extension project)

For some reasons, ESLint stops checking my code. I tried various steps to make ESLint work again, but it only worsens the problem as shown below:

![image](https://user-images.githubusercontent.com/91402471/170910700-d8be676a-8884-4554-9673-8f8fccaef77e.png)

![image](https://user-images.githubusercontent.com/91402471/170910724-9cb09a88-6ca7-4cfb-92f3-e2b789fe88fe.png)

Is package.json a requirement on new version of ESLint now?

**Comment by dbaeumer** [comment]
Any steps how to reproduce this, best via a GitHub repository I can clone. 

I tried and the command works for me or shows the following notification in the UI:
<img width="356" alt="capture" src="https://user-images.githubusercontent.com/1931590/171007537-48cd7c58-e100-4406-ba0a-7770ee9b0704.png">

**Comment by dbaeumer** [comment]
OK. Thanks.

**Comment by khanguslee** [comment]
> Problem solved, steps:
> 
> Uninstalled global eslint from npm
> 
> Installed eslint from brew
> 
> Re-created eslint configuration

Extending upon this, I had to run `yarn global remove eslint` and this worked for me!

**Comment by dbaeumer** [comment]
I will close the issue. reading through the comments again I don't see any action that needs to be taken in the ESLint extension.

**Comment by JorensM** [comment]
I'm getting this and also my ESLint extension is not working, not showing me warnings etc. I don't have ESLint installed globally anywhere, just as a VSCode extension.

**Comment by Paovilla** [comment]
I was getting the same issue, and after trying many solutions, updating vscode works!! thank you so much

**Comment by kaspar-p** [comment]
In my case, it was _downgrading_ the extension version to match my (necessarily) old VSCode version.

**Comment by peters-finstreet** [comment]
For me the issue was similar to @JorensM's issue. Probably since a recent update of eslint the minimal version of VS code was raised which you could see in the ESlint output panel (View => Appearance => Panel => "Output" tab in the panel => select ESlint in the top right).
VS code just didn't update automatically because I had installed it in the downloads folder on Mac OS and there seems to be some kind of restriction being put on applications by [Squirrel](https://github.com/microsoft/vscode/issues/7426#issuecomment-425093469)

TLDR: ESlint raised it's supported VS code version. Should be fixed by updating VS code

**Comment by dan1xk** [comment]
For me it worked by updating vscode, I ran this command on windows: 

> winget upgrade --id Microsoft.VisualStudioCode
